### PR TITLE
fix exception call stack in load.

### PR DESCRIPTION
**Pull request by vwxyzh**
@chenkennt @ansyral @hellosnow @superyyrrzz @qinezh

### CODE REVIEW

File: src/Microsoft.DocAsCode.Build.Engine/SingleDocumentBuilder.cs [hunk 1]
```diff
@@ -220,7 +220,7 @@ private void Cleanup(HostService hostService)
             }
             catch (AggregateException ex)
             {
-                throw ex.InnerException;
+                throw new DocfxException(ex.InnerException?.Message, ex);
```
Comment (hellosnow) [inline src/Microsoft.DocAsCode.Build.Engine/SingleDocumentBuilder.cs]: >ex [](start = 69, length = 2)

ex.InnerException?

**Comment by hellosnow** [comment]
:shipit:

**Comment by superyyrrzz** [comment]
:shipit:

File: src/Microsoft.DocAsCode.Build.Engine/SingleDocumentBuilder.cs (hunk 1 above)
Comment (vwxyzh) [inline src/Microsoft.DocAsCode.Build.Engine/SingleDocumentBuilder.cs]: If multi exceptions occurs, we should give all exceptions.

**Pull request merged**